yarn.lock

```diff
@@ -9319,9 +9319,9 @@ performance-now@^2.1.0:
   resolved "https://registry.yarnpkg.com/performance-now/-/performance-now-2.1.0.tgz#6309f4e0e5fa913ec1c69307ae364b4b377c9e7b"
   integrity sha1-Ywn04OX6kT7BxpMHrjZLSzd8nns=
 
-"pi-ui@https://github.com/decred/pi-ui":
+"pi-ui@https://github.com/amassarwi/pi-ui#checkboxAdj":
   version "1.0.0"
-  resolved "https://github.com/decred/pi-ui#c9741462323ab7d7c30059827f8b618f7bfc5b8d"
+  resolved "https://github.com/amassarwi/pi-ui#b124cc03cc5087a533a86850dd9f5a96b092250b"
```

Comment:
Will revert when pi-ui's PR is in

Reply:
Hrm would prefer to wait on this one until pi-ui is up to date.

Reply:
of course, commented to remind myself what to revert before merging.

Reply:
gotcha, looks like it's failing nonetheless

Reply:
yep, rebase smoke 🚒 🧯 🧑‍🚒